packages/palette/src/elements/Select/Select.test.tsx

```diff
@@ -23,6 +35,16 @@ describe("Select", () => {
       expect(wrapper.find("select").props().value).toBe("secondOption")
     })
 
+    it("renders with a placeholder value", () => {
+      const wrapper = mount(
+        <SelectSmall options={options} placeholder="I'm a placeholder" />
+      )
+      expect(wrapper.find("select").text()).toMatch(
+        new RegExp("^I'm a placeholder")
```

Comment:
Why use  a regex here?

Reply:
I should probably do something better- this was just from copying the adjacent tests around.